pkgs/development/tools/misc/universal-ctags/default.nix

```diff
@@ -2,22 +2,25 @@
 
```

Comment:
@teto Which is your opinion about add libxml2, jansson and libyaml dependencies?

Reply:
What are the benefits ? if it's to export the data in some format, I would say let it be. The interface doesn't seem too stable yet.